### PR TITLE
test(server): how to re-create the BigQuery test DB

### PR DESCRIPTION
**Pull request by davinov**

### CODE REVIEW

**Comment by davinov** [comment]
Actually the auto-detection of the schema doesn't seem to work

I tried also with 
```[
    {
        "name": "price_per_l",
        "type": "NUMERIC",
        "mode": "REQUIRED"
    },
    {
        "name": "alcohol_degree",
        "type": "NUMERIC",
        "mode": "REQUIRED"
    },
    {
        "name": "name",
        "type": "STRING",
        "mode": "REQUIRED"
    },
    {
        "name": "cost",
        "type": "NUMERIC",
        "mode": "REQUIRED"
    },
    {
        "name": "beer_kind",
        "type": "STRING",
        "mode": "REQUIRED"
    },
    {
        "name": "volume_ml",
        "type": "INTEGER",
        "mode": "REQUIRED"
    },
    {
        "name": "brewing_date",
        "type": "DATE",
        "mode": "REQUIRED"
    },
    {
        "name": "nullable_name",
        "type": "STRING",
        "mode": "NULLABLE"
    }
]
```
but it looks like some values are not parsable as numerics:
`Failed to create table: Error while reading data, error message: Could not parse '4.877377162629759' as NUMERIC for field alcohol_degree (position 1) starting at location 139 with message 'Invalid NUMERIC value: 4.877377162629759`

**Comment by lukapeschke** [comment]
[gbq_query.txt](https://github.com/ToucanToco/weaverbird/files/12145753/gbq_query.txt)
[gbq_schema.txt](https://github.com/ToucanToco/weaverbird/files/12145754/gbq_schema.txt)
[beers-bigquery.csv](https://github.com/ToucanToco/weaverbird/files/12145757/beers-bigquery.csv)

The `beers` table  should be created from the `beers-bigquery.csv` file, with the schema in `gbq_schema.txt`. The `beers_tiny` can then be created by manually running the query in `gbq_query.txt`. If I remember correctly, @jGundermann had written that down somewhere in confluence

**Comment by davinov** [comment]
I couldn't find it with the Confluence search :/
I'll add it here in the comments so it's easier to find :)

File: server/tests/backends/sql_translator_integration_tests/test_sql_bigquery_translator_steps.py [hunk 1]
```diff
@@ -1,3 +1,22 @@
+"""
+BigQuery free DBs have tables that expire after 60 days.
+If the table "beers.beers_tiny" is expired, re-create it:
+- open the BigQuery console https://console.cloud.google.com/bigquery?project=biquery-integration-tests&ws=!1m4!1m3!3m2!1sbiquery-integration-tests!2sbeers
+- use "create table", choose "Upload" and use the `beers-biugquery.csv` file available [here](https://github.com/ToucanToco/weaverbird/pull/1835#issuecomment-1647810149)
```
Comment (lukapeschke) [inline server/tests/backends/sql_translator_integration_tests/test_sql_bigquery_translator_steps.py:5]:
```suggestion
- use "create table", choose "Upload" and use the `beers-bigquery.csv` file available [here](https://github.com/ToucanToco/weaverbird/pull/1835#issuecomment-1647810149)
```

File: server/tests/backends/sql_translator_integration_tests/test_sql_bigquery_translator_steps.py [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+    price_per_l:FLOAT,
+    alcohol_degree:FLOAT,
+    name:STRING,
+    cost:FLOAT,
+    beer_kind:STRING,
+    volume_ml:FLOAT,
+    brewing_date:DATE,
+    nullable_name:STRING
+    ```
+- run the query: CREATE TABLE `beers.beers_tiny` AS SELECT * FROM `beers.beers` ORDER BY brewing_date LIMIT 10
```
Comment (lukapeschke) [inline server/tests/backends/sql_translator_integration_tests/test_sql_bigquery_translator_steps.py:18]: Maybe the whole query could be wrapped in triple quotes ?

**Review by lukapeschke — COMMENTED** [review]
LGTM! Just a typo and a suggestion

**Comment by davinov** [comment]
Good points! Fixed in [ed9be97](https://github.com/ToucanToco/weaverbird/pull/1835/commits/ed9be97aa3f784dc3df950595d637e1074e834cf)

**Review by lukapeschke — APPROVED** [review]
:ok_hand:

**Pull request merged**